src/openrct2-win/openrct2-win.cpp

```diff
@@ -17,12 +17,16 @@
     "\"/manifestdependency:type='win32' name='Microsoft.Windows.Common-Controls' version='6.0.0.0' processorArchitecture='*' publicKeyToken='6595b64144ccf1df' language='*'\"")
 
 // Then the rest
+#include <memory>
 #include <openrct2-ui/Ui.h>
 #include <stdio.h>
 #include <stdlib.h>
 
 static char** GetCommandLineArgs(int argc, wchar_t** argvW);
-static void FreeCommandLineArgs(int argc, char** argv);
+
+//-------removed as manually deallocating the memory is no longer required as using smart ptrs  ,BY VS
```

Comment:
We have source control to keep track of removed code. Just remove it. Also, no need to graffiti your initials all over the place.

Reply:
Sure , thought of it , just have a habit of doing it  .

Reply:
I have done the changes as asked